### PR TITLE
fix: [CDS-41559]: style fixes + mini support added to connector field for variable section

### PR DESCRIPTION
**Pull request by PratyushGar**
#### Screenshots

image

image

image

All connectors allowed
image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 #### 5 Issues 
* Code issue 5 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11173)

**Review by yc147 — APPROVED** [review]

**Review by satyam-harness — DISMISSED** [review]

**Review by yc147 — APPROVED** [review]

**Review by yc147 — APPROVED** [review]

File: src/modules/35-connectors/components/ConnectorReferenceField/FormMultiTypeConnectorField.tsx [hunk 1]
```diff
@@ -435,8 +437,13 @@ export const MultiTypeConnectorField = (props: MultiTypeConnectorFieldProps): Re
...
   return (
-    <div style={style} className={cx(css.connectorLabel, connectorLabelClassFromProps)}>
-      <Container style={{ marginBottom: 5 }}>
+    <div
+      style={style}
+      className={cx(css.connectorLabel, connectorLabelClassFromProps, {
+        [css.mini]: mini
+      })}
+    >
+      <Container style={{ marginBottom: 5 }} data-test-id="connectorTooltip">
```
Comment (reetika04) [inline src/modules/35-connectors/components/ConnectorReferenceField/FormMultiTypeConnectorField.tsx:446]: Can we avoid using inline styles and use classnames??

Comment (PratyushGar) [reply src/modules/35-connectors/components/ConnectorReferenceField/FormMultiTypeConnectorField.tsx:446]: Agreed, this already existed, will update in the follow-up PR.

File: src/modules/75-cd/components/TemplateStudio/DeploymentTemplateCanvas/DeploymentTemplateForm/DeploymentInfraWrapper/DeploymentInfraUtils.tsx [hunk 2]
```diff
@@ -47,10 +49,27 @@ export function getValidationSchema(getString: UseStringsReturn['getString']): Y
     instanceAttributes: Yup.array()
       .of(
         Yup.object().shape({
-          name: Yup.string().required(getString('common.validation.nameIsRequired')),
+          name: Yup.string()
```
Comment (reetika04) [inline src/modules/75-cd/components/TemplateStudio/DeploymentTemplateCanvas/DeploymentTemplateForm/DeploymentInfraWrapper/DeploymentInfraUtils.tsx:52]: We already have NameSchema, can we reuse that? If not, please add this to the common files.

**Review by reetika04 — APPROVED** [review]

**Review by YounixIO — APPROVED** [review]

**Pull request merged**